common/common.cc

```diff
@@ -179,7 +179,8 @@ sorbet::FileOps::ReadLineOutput sorbet::FileOps::readLineFromFd(int fd, string &
     }
 
     constexpr int BUFF_SIZE = 1024 * 8;
-    vector<char> buf(BUFF_SIZE);
+    char chars[BUFF_SIZE];
+    absl::Span<char> buf(chars);
```

Comment:
Huh. What happens if there’s more to read?

Reply:
We buffer up what we just read and "timeout":

https://github.com/sorbet/sorbet/blob/8f47605b1a4327a90ccebc621be511f660844e5e/common/common.cc#L192-L208